website/docs/core-concepts/writing-code/javascript-promises.md

```diff
@@ -212,3 +212,44 @@ In the preceding example:
 1. Run `MockApi` query with the parameter `name` as 'Appsmith' and wait for the response.
 2. Store the response in the Appsmith store using `storeValue()` when you get the response.
 3. On successful execution of `storeValue()`, show an alert message with the data saved in the store.
+
+
+## Fetch API
+The Fetch API provides an interface for executing network calls programmatically. You can use `fetch()` to programmatically configure and execute a REST API.  
+
+**Examples**
+
+**GET request**
+```javascript
+await fetch("https://opentdb.com/api.php?amount=10")
```

Comment:
I think this should be
`const questions = await fetch("https://opentdb.com/api.php?amount=10")`